packages/jazz-tools/src/svelte/media/image.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+import { type FileStream, ImageDefinition } from "jazz-tools";
+import { highestResAvailable } from "jazz-tools/media";
+import { onDestroy } from "svelte";
+import type { HTMLImgAttributes } from "svelte/elements";
+import { CoState } from "../jazz.class.svelte";
+
+interface ImageProps extends Omit<HTMLImgAttributes, "width" | "height"> {
+  imageId: string;
+  width?: number | "original";
+  height?: number | "original";
+}
+
+const { imageId, width, height, ...rest }: ImageProps = $props();
+
+// Create reactive state for the image
+const imageState = new CoState(ImageDefinition, imageId);
+
+let src: string | undefined = $state(imageState.current?.placeholderDataURL);
+
+const dimensions = $derived.by<{
+  width: number | undefined;
+  height: number | undefined;
+}>(() => {
+  const w =
+    width === "original" ? imageState.current?.originalSize?.[0] : width;
+  const h =
+    height === "original" ? imageState.current?.originalSize?.[1] : height;
+
+  return { width: w, height: h };
+});
+
+$effect(() => {
+  const image = imageState.current;
+  if (!image) return;
+
+  src = image.placeholderDataURL;
+  let lastBestImage: FileStream | null = null;
+
+  const unsub = image.subscribe({}, (update) => {
```

Comment:
Same as in React.

I think it would be better to subscribe from the id for a more stable reactivity on the effect